lib/log_parser.rb

```diff
@@ -0,0 +1,32 @@
+class LogParser
+    def initialize(args)
+      if args.first.nil?
```

Comment:
Se é obrigatório um parametro/argumento, evite utilizar algo como "args", caso queira deixar como opcional, vc pode fazer algo como:
```suggestion
  def initialize(file_path = "games.log")
      @file_name = file_path
```

Reply:
Entendi, usei essa dica, corrigi aqui.